.github/workflows/limit-pr-files-changed.yml

```diff
@@ -0,0 +1,70 @@
+name: Enforce Max Lines Changed Per File
+
+on:
+  pull_request:
+    paths:
+      - '**/*.java'  # Monitor changes to Java files only
```

Comment:
Can we also include `.avsc` and `.proto` files please?

Schema changes ideally deserve dedicated PRs anyway, but without making things too complicated at first, we could at least include them in the count?

Reply:
But for new protocol updates, 'added' will count the entire file as added which would be very high most of the time. Do we ever do inplace avsc changes?

Reply:
Well that is the point, protocol changes should be reviewed in isolation, prior to any code getting written 😁